docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -343,15 +343,15 @@ export default {
     ).href;
 
     return (
-      <Zoom>
-        <div className="mx-auto">
+      <Zoom wrapElement="span" wrapStyle={{display: 'block'}}>
+        <span className="block mx-auto">
           <Image
             src={resolvedPath}
             width={props.width}
             height={props.height}
             alt={props.alt}
           />
-        </div>
+        </span>
       </Zoom>
     );
```

Comment:
Fixed this error while I'm here. This error often leads to tricky SSR problems that are really hard to debug (because it might only repro in production builds), so it's worth giving it some attention

<img width="760" alt="image" src="https://user-images.githubusercontent.com/2268452/154758231-d9ebafac-fbf0-4532-9e8b-23c39ea92939.png">